ajax/libs/jquery.loadTemplate/package.json

```diff
@@ -0,0 +1,26 @@
+{
+    "name": "jquery.loadTemplate",
+    "description": "A template plugin for jQuery. Allows templating without cluttering JavaScript code with markup",
+    "filename": "jquery.loadTemplate.min.js",
+    "keywords": [
+       "templates",
+       "templating",
+       "jquery-plugin"
+    ],
+    "author": "codepb",
```

Comment:
Let's let the author as same with upstream.
modify to `"author": "Paul Burgess",`

Here is the upstream's package.json file you could refer to https://github.com/codepb/jquery-template/blob/master/package.json#L5

Thank you.